---
int32 result
int64 queuedCmdCurrentIndex
